react-native/react/native/remote-manager.desktop.js

```diff
@@ -15,19 +16,26 @@ import type {Action, Dispatch} from '../constants/types/flux'
 
 import type {TrackerState} from '../reducers/tracker'
 import type {PinentryState} from '../reducers/pinentry'
+import type {ShowUpdateState} from '../reducers/update'
 
 export type RemoteManagerProps = {
   registerPinentryListener: () => void,
+  registerUpdateListener: () => void,
   pinentryOnCancel: (sessionID: number) => void,
   pinentryOnSubmit: (sessionID: number, passphrase: string, features: GUIEntryFeatures) => void,
   registerIdentifyUi: () => void,
   registerTrackerChangeListener: () => void,
   onCloseFromHeader: () => void,
   trackerServerStarted: boolean,
-  startTimer: (dispatch: Dispatch, getState: any) => void,
-  stopTimer: () => Action,
+  trackerStartTimer: (dispatch: Dispatch, getState: any) => void,
+  trackerStopTimer: () => Action,
   trackers: {[key: string]: TrackerState},
-  pinentryStates: {[key: string]: PinentryState}
+  pinentryStates: {[key: string]: PinentryState},
+  showUpdateState: ShowUpdateState,
+  updateOnSkip: () => void,
```

Comment:
how this thing works is kinda wonky but let's leave it as-is for now. we should at some point generalize this so we commonize some redundant code and don't have to put all these props directty in here. we should have sub props that are just for the component itself and not this flat thing